From b35f09e81d292d3cfbe0d57db17550661f2b8800 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 28 May 2024 14:22:23 +0200
Subject: [PATCH 0910/1695] dts: stm32: remove include of ETZPC bindings in
 stm32mp15x dt files

Since it is included at SoC level, remove all includes of these bindings
at board level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f2d58a7ee44e459cd00fd6baae2894529960543c dts: stm32: remove include of ETZPC bindings in stm32mp15x dt files]
Change-Id: I994dc7cdbad56095d06989eebaacc1a788ddc6bf
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/383370
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp157a-dk1.dts  | 1 -
 core/arch/arm/dts/stm32mp157a-ed1.dts  | 1 -
 core/arch/arm/dts/stm32mp157a-ev1.dts  | 1 -
 core/arch/arm/dts/stm32mp157c-ed1.dts  | 1 -
 core/arch/arm/dts/stm32mp157c-ev1.dts  | 2 --
 core/arch/arm/dts/stm32mp157d-ed1.dts  | 1 -
 core/arch/arm/dts/stm32mp157d-ev1.dts  | 1 -
 core/arch/arm/dts/stm32mp157f-ed1.dts  | 1 -
 core/arch/arm/dts/stm32mp157f-ev1.dts  | 1 -
 core/arch/arm/dts/stm32mp15xx-dkx.dtsi | 1 -
 10 files changed, 11 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp157a-dk1.dts b/core/arch/arm/dts/stm32mp157a-dk1.dts
index c12c1c8a1..9c43e8411 100644
--- a/core/arch/arm/dts/stm32mp157a-dk1.dts
+++ b/core/arch/arm/dts/stm32mp157a-dk1.dts
@@ -6,7 +6,6 @@
 
 /dts-v1/;
 
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 #include "stm32mp157.dtsi"
 #include "stm32mp15xa.dtsi"
 #include "stm32mp15-pinctrl.dtsi"
diff --git a/core/arch/arm/dts/stm32mp157a-ed1.dts b/core/arch/arm/dts/stm32mp157a-ed1.dts
index bf89943aa..86ad9a06f 100644
--- a/core/arch/arm/dts/stm32mp157a-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157a-ed1.dts
@@ -10,7 +10,6 @@
 #include <dt-bindings/mfd/st,stpmic1.h>
 #include <dt-bindings/power/stm32mp1-power.h>
 #include <dt-bindings/soc/stm32mp-provisioning.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 
 #include "stm32mp157.dtsi"
 #include "stm32mp15xa.dtsi"
diff --git a/core/arch/arm/dts/stm32mp157a-ev1.dts b/core/arch/arm/dts/stm32mp157a-ev1.dts
index fc890b279..fad6da915 100644
--- a/core/arch/arm/dts/stm32mp157a-ev1.dts
+++ b/core/arch/arm/dts/stm32mp157a-ev1.dts
@@ -8,7 +8,6 @@
 
 #include <dt-bindings/clock/stm32mp1-clksrc.h>
 #include <dt-bindings/mfd/st,stpmic1.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 
 #include "stm32mp157a-ed1.dts"
 
diff --git a/core/arch/arm/dts/stm32mp157c-ed1.dts b/core/arch/arm/dts/stm32mp157c-ed1.dts
index 7d8178075..f35e2cf97 100644
--- a/core/arch/arm/dts/stm32mp157c-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157c-ed1.dts
@@ -10,7 +10,6 @@
 #include <dt-bindings/mfd/st,stpmic1.h>
 #include <dt-bindings/power/stm32mp1-power.h>
 #include <dt-bindings/soc/stm32mp-provisioning.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 
 #include "stm32mp157.dtsi"
 #include "stm32mp15xc.dtsi"
diff --git a/core/arch/arm/dts/stm32mp157c-ev1.dts b/core/arch/arm/dts/stm32mp157c-ev1.dts
index 82eb20c67..82ed596aa 100644
--- a/core/arch/arm/dts/stm32mp157c-ev1.dts
+++ b/core/arch/arm/dts/stm32mp157c-ev1.dts
@@ -5,8 +5,6 @@
  */
 /dts-v1/;
 
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
-
 #include "stm32mp157c-ed1.dts"
 #include <dt-bindings/gpio/gpio.h>
 
diff --git a/core/arch/arm/dts/stm32mp157d-ed1.dts b/core/arch/arm/dts/stm32mp157d-ed1.dts
index 7376f9ad7..b641b0f4e 100644
--- a/core/arch/arm/dts/stm32mp157d-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157d-ed1.dts
@@ -10,7 +10,6 @@
 #include <dt-bindings/mfd/st,stpmic1.h>
 #include <dt-bindings/power/stm32mp1-power.h>
 #include <dt-bindings/soc/stm32mp-provisioning.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 
 #include "stm32mp157.dtsi"
 #include "stm32mp15xd.dtsi"
diff --git a/core/arch/arm/dts/stm32mp157d-ev1.dts b/core/arch/arm/dts/stm32mp157d-ev1.dts
index e41931228..1ad446a3f 100644
--- a/core/arch/arm/dts/stm32mp157d-ev1.dts
+++ b/core/arch/arm/dts/stm32mp157d-ev1.dts
@@ -8,7 +8,6 @@
 
 #include <dt-bindings/clock/stm32mp1-clksrc.h>
 #include <dt-bindings/mfd/st,stpmic1.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 
 #include "stm32mp157d-ed1.dts"
 
diff --git a/core/arch/arm/dts/stm32mp157f-ed1.dts b/core/arch/arm/dts/stm32mp157f-ed1.dts
index bcf3daf47..3e3a517cd 100644
--- a/core/arch/arm/dts/stm32mp157f-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157f-ed1.dts
@@ -10,7 +10,6 @@
 #include <dt-bindings/mfd/st,stpmic1.h>
 #include <dt-bindings/power/stm32mp1-power.h>
 #include <dt-bindings/soc/stm32mp-provisioning.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 
 #include "stm32mp157.dtsi"
 #include "stm32mp15xf.dtsi"
diff --git a/core/arch/arm/dts/stm32mp157f-ev1.dts b/core/arch/arm/dts/stm32mp157f-ev1.dts
index d7556d99a..8992a9f34 100644
--- a/core/arch/arm/dts/stm32mp157f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp157f-ev1.dts
@@ -8,7 +8,6 @@
 
 #include <dt-bindings/clock/stm32mp1-clksrc.h>
 #include <dt-bindings/mfd/st,stpmic1.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 
 #include "stm32mp157f-ed1.dts"
 
diff --git a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
index 5d4144030..4948dd27f 100644
--- a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
+++ b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
@@ -9,7 +9,6 @@
 #include <dt-bindings/power/stm32mp1-power.h>
 #include <dt-bindings/mfd/st,stpmic1.h>
 #include <dt-bindings/soc/stm32mp-provisioning.h>
-#include <dt-bindings/soc/stm32mp15-etzpc.h>
 #include <dt-bindings/tamper/st,stm32-tamp.h>
 
 / {
-- 
2.47.2

